docs/src/modules/javascript/pages/proto.adoc

```diff
@@ -95,6 +106,15 @@ It includes the messages that define the entity state:
 include::example$js-valueentity-shopping-cart/proto/shoppingcart_domain.proto[tag=proto_messages]
 ----
 
+To use a multi-field key, add the `entity_key` notation to each field. For example, if the service API proto file `SomeMessage` element defines both `first_field` and `second_field` as part of the key, do the same for the domain message:
+
+[source,proto]
+----
+message SomeMessage {
+    string first_field = 1 [(akkaserverless.field).entity_key = true];
+    string second_field = 2 [(akkaserverless.field).entity_key = true];
+----
+
```

Comment:
Entity keys are not defined on domain messages. This part can be removed.

```suggestion
```

Reply:
Done.